openedx/core/djangoapps/course_groups/partition_scheme.py

```diff
@@ -4,7 +4,11 @@
 import logging
 
 from courseware import courses
-from courseware.masquerade import get_masquerading_group_info
+from courseware.masquerade import (  # pylint: disable=import-error
```

Comment:
I'm not sure why this `pylint` directive is necessary. Without it, pylint [was failing](https://build.testeng.edx.org/job/edx-platform-quality-pr/803/console) on Jenkins with:

```
...
07:45:03 openedx/core/djangoapps/course_groups/partition_scheme.py (92.9%):
07:45:03     7: F0401: (import-error), : Unable to import 'courseware.masquerade'
07:45:03 -------------
07:45:03 Total:   68 lines
07:45:03 Violations: 1 line
07:45:03 % Quality: 98%
07:45:03 -------------
```
even though `from courseware.masquerade import ...` was already in use here, and the `run_quality` check was passing for me on my devstack.